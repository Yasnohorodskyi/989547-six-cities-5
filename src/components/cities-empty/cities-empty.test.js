import React from "react";
import renderer from "react-test-renderer";

import CitiesEmpty from "./cities-empty";

it(`Should render main block of main page with no cities`, () => {
  const tree = renderer
  .create(<CitiesEmpty />).toJSON();

  expect(tree).toMatchSnapshot();
});
