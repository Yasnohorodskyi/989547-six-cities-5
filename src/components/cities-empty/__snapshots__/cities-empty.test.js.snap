// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render main block of main page with no cities 1`] = `
<section
  className="cities__no-places"
>
  <div
    className="cities__status-wrapper tabs__content"
  >
    <b
      className="cities__status"
    >
      No places to stay available
    </b>
    <p
      className="cities__status-description"
    >
      We could not find any property available at the moment in Dusseldorf
    </p>
  </div>
</section>
`;
